src-java/kilda-model/src/main/java/org/openkilda/model/HaFlowPath.java

```diff
@@ -175,11 +160,19 @@ public void setSharedBandwidthGroupId(String sharedBandwidthGroupId) {
      * Shared switch id is not included into this set.
      */
     public Set<SwitchId> getSubFlowSwitchIds() {
-        return getHaSubFlowEdges().stream()
-                .map(HaSubFlowEdge::getSubFlowEndpointSwitchId)
+        return getHaSubFlows().stream()
+                .map(HaSubFlow::getEndpointSwitchId)
                 .collect(Collectors.toSet());
     }
 
+    /**
+     * Check whether the path is protected for the flow.
```

Comment:
The same as above, slightly ambiguous. Probably better to use `checks whether this path is a protected path for the flow`.